kaskade/main.py

```diff
@@ -79,10 +79,22 @@ def cli() -> None:
         metavar="filename",
     ),
 )
+@cloup.option_group(
+    "Cloud configuration options",
```

Comment:
I would prefer AWS, and instead of `--cloud-config` we pass `--aws`, so the very optional options will be: 

```
--registry
--avro
--protobuf
--aws
```